package.json

```diff
@@ -2,27 +2,515 @@
+    "spdx-correct": "^3.1.1",
+    "source-map-support": "^0.5.21",
+    "spdy-transport": "^3.0.0",
+    "spdy": "^4.0.2",
+    "spdx-license-ids": "^3.0.12",
+    "sprintf-js": "^1.0.3",
+    "spdx-expression-parse": "^3.0.1",
+    "statuses": "^2.0.1",
+    "string_decoder": "^1.3.0",
+    "strip-bom": "^3.0.0",
+    "specificity": "^0.4.1",
+    "string.prototype.trimend": "^1.0.6",
+    "strip-json-comments": "^3.1.1",
+    "strip-final-newline": "^2.0.0",
+    "strip-indent": "^3.0.0",
+    "style-loader": "^3.3.1",
+    "string-width": "^4.2.3",
+    "string.prototype.trimstart": "^1.0.6",
+    "style-search": "^0.1.0",
+    "strip-ansi": "^6.0.1",
+    "stylelint-config-recommended": "^4.0.0",
+    "stylelint-config-standard": "^21.0.0",
+    "sugarss": "^2.0.0",
+    "stylelint": "^13.13.1",
+    "stylelint-scss": "^3.21.0",
+    "supports-preserve-symlinks-flag": "^1.0.0",
+    "supports-color": "^8.1.1",
+    "svg-tags": "^1.0.0",
+    "text-table": "^0.2.0",
+    "terser-webpack-plugin": "^5.3.6",
+    "table": "^6.8.1",
+    "stylelint-csstree-validator": "^1.9.0",
+    "send": "^0.18.0",
+    "tapable": "^2.2.1",
+    "terser": "^5.16.2",
+    "trim-newlines": "^3.0.1",
+    "trough": "^1.0.5",
+    "to-regex-range": "^5.0.1",
+    "type-fest": "^0.20.2",
+    "to-fast-properties": "^2.0.0",
+    "toidentifier": "^1.0.1",
+    "typedarray-to-buffer": "^3.1.5",
+    "type-is": "^1.6.18",
+    "thunky": "^1.1.0",
+    "unified": "^9.2.2",
+    "tslib": "^2.5.0",
+    "unist-util-find-all-after": "^3.0.2",
+    "unist-util-is": "^4.1.0",
+    "typed-array-length": "^1.0.4",
+    "unpipe": "^1.0.0",
+    "type-check": "^0.4.0",
+    "tsconfig-paths": "^3.14.1",
+    "update-browserslist-db": "^1.0.10",
+    "uri-js": "^4.4.1",
+    "utila": "^0.4.0",
+    "util-deprecate": "^1.0.2",
+    "utils-merge": "^1.0.1",
+    "vary": "^1.1.2",
+    "vfile": "^4.2.1",
+    "watchpack": "^2.4.0",
+    "wbuf": "^1.7.3",
+    "unist-util-stringify-position": "^2.0.3",
+    "v8-compile-cache": "^2.3.0",
+    "webpack": "^5.75.0",
+    "validate-npm-package-license": "^3.0.4",
+    "webpack-merge": "^5.8.0",
+    "webpack-dev-server": "^4.11.1",
+    "webpack-sources": "^3.2.3",
+    "websocket-extensions": "^0.1.4",
+    "uuid": "^8.3.2",
+    "which": "^2.0.2",
+    "vfile-message": "^2.0.4",
+    "webpack-dev-middleware": "^5.3.3",
+    "webpack-cli": "^5.0.1",
+    "websocket-driver": "^0.7.4",
+    "wildcard": "^2.0.0",
+    "wrappy": "^1.0.2",
+    "which-boxed-primitive": "^1.0.2",
+    "word-wrap": "^1.2.3",
+    "unbox-primitive": "^1.0.2",
+    "which-typed-array": "^1.1.9",
+    "write-file-atomic": "^3.0.3",
+    "yaml": "^1.10.2",
+    "yargs-parser": "^20.2.9",
+    "zwitch": "^1.0.5",
+    "yallist": "^4.0.0",
+    "ws": "^8.12.0"
+  },
+  "devDependencies": {},
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "webpack"
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
- Your package.json should have  scripts like   `start` to enable you start the project with `npm start`  and enable live reloading,  please see [webpack documentation](https://webpack.js.org/guides/development/#using-webpack-dev-server) to see how to set it up  👍🏿